lib/prepare_release.js

```diff
@@ -67,9 +72,144 @@ export default class ReleasePreparation {
+        continue;
+      }
+      const cp = new CherryPick(pr.number, this.dir, cli, {
+        owner: this.owner,
+        repo: this.repo,
+        lint: false,
+        includeCVE: true
+      });
+      const success = await cp.start();
+      if (!success) {
+        cli.warn('The cherry-pick has failed. PR-ID: ' + pr.number);
+        return false;
+      }
+    }
+    return true;
+  }
+
+  async prepareSecurity() {
+    const {
+      cli,
+      newVersion,
+      versionComponents,
+      releaseBranch,
+      filterLabels
+    } = this;
+
+    // Create new proposal branch.
+    cli.startSpinner(`Creating new proposal branch for ${newVersion}`);
+    const proposalBranch = await this.createProposalBranch(releaseBranch);
+    cli.stopSpinner(`Created new proposal branch for ${newVersion}`);
+
+    const success = await this.cherryPickSecurityPRs(filterLabels);
+    if (!success) {
+      cli.error('Aborting security release preparation. ' +
+        'Remember to exclude the proposal branch.' +
+        'git branch -D ' + proposalBranch);
+      return;
+    }
+    // Update version and release info in src/node_version.h.
+    cli.startSpinner(`Updating 'src/node_version.h' for ${newVersion}`);
+    await this.updateNodeVersion();
+    cli.stopSpinner(`Updated 'src/node_version.h' for ${newVersion}`);
+
+    // Update any REPLACEME tags in the docs.
+    cli.startSpinner('Updating REPLACEME items in docs');
+    await this.updateREPLACEMEs();
+    cli.stopSpinner('Updated REPLACEME items in docs');
+
+    // Fetch date to use in release commit & changelogs.
+    const todayDate = new Date().toISOString().split('T')[0];
+    this.date = await cli.prompt('Enter release date in YYYY-MM-DD format:',
+      { questionType: 'input', defaultAnswer: todayDate });
+
+    cli.startSpinner('Updating CHANGELOG.md');
+    await this.updateMainChangelog();
+    cli.stopSpinner('Updated CHANGELOG.md');
+
+    cli.startSpinner(`Updating CHANGELOG_V${versionComponents.major}.md`);
+    await this.updateMajorChangelog();
+    cli.stopSpinner(`Updated CHANGELOG_V${versionComponents.major}.md`);
+
+    await cli.prompt('Finished editing the changelogs?',
+      { defaultAnswer: false });
+
+    // Create release commit.
+    const shouldCreateReleaseCommit = await cli.prompt(
+      'Create release commit?');
+    if (!shouldCreateReleaseCommit) {
+      cli.warn(`Aborting \`git node release\` for version ${newVersion}`);
+      return;
+    }
+
+    // Proceed with release only after the releaser has amended
+    // it to their liking.
+    const createDefaultCommit = await this.createReleaseCommit();
+    if (!createDefaultCommit) {
+      const lastCommitSha = runSync('git', ['rev-parse', '--short', 'HEAD']);
+      cli.warn(`Please manually edit commit ${lastCommitSha} by running ` +
+      '`git commit --amend` before proceeding.');
+
+      await cli.prompt(
+        'Finished editing the release commit?',
+        { defaultAnswer: false });
```

Comment:
does `cli.prompt` throws if the answer is negative or something? I'm not sure what this prompt here is doing either 🤔

Reply:
it was just a copy-paste from the regular release process :). Fixed, thanks